### PR TITLE
Multiple v2 fixes

### PR DESCRIPTION
**Pull request by theresakhoo**
Fixes for
- allow encoders to handle both string and object parts
- when generating the MNFv1 resource, set both str and nstr
- set cost attribution for TranslationOS translator

### CODE REVIEW

File: helpers/src/normalizers.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ export const doublePercentDecoder = decoderMaker(
 export function gatedEncoder(encoder, ...flagNames) {
     const fn = function gatedEncoder(str, flags = {}) {
         const run = flagNames.reduce((run, flag) => run || (flag.charAt(0) === '!' ? !flags[flag.substring(1)] : flags[flag]), false);
+        str = typeof str === 'string' ? str : str.v;
```
Comment (diegone) [inline helpers/src/normalizers.js:26]: this prevents the `encoder()` to receive the full object. i think this is only needed when `run` is false.

File: package.json [hunk 2]
```diff
@@ -10,6 +10,9 @@
             "js"
         ]
     },
+    "scripts": {
+        "test": "node node_modules/jest/bin/jest.js"
```
Comment (diegone) [inline package.json:14]: why do we need this? tests used to be run from the /regression and /test directories. if we want to launch module-level tests, we can use workspaces.

Comment (theresakhoo) [reply package.json:14]: Was debating whether to keep this or not.  Removed.

File: helpers/src/normalizers.js (hunk 1 above)
Comment (theresakhoo) [reply helpers/src/normalizers.js:26]: Changed as suggested

File: helpers-translated/translationOS.js [hunk 3]
```diff
@@ -108,7 +108,8 @@ module.exports = class TranslationOS {
                 'user-agent': 'l10n.monster/TOS v0.1',
                 'Content-Type': 'application/json',
             }
-            this.serviceType = serviceType ?? 'premium',
+            this.serviceType = serviceType ?? 'premium';
+            this.costAttributionLabel = costAttributionLabel;
```
Comment (diegone) [inline helpers-translated/translationOS.js:112]: this is not used anywhere

**Pull request merged**